src/Editor.cpp

```diff
@@ -838,7 +844,41 @@ float Editor::GetScaleUI() const {
     return  _settings._uiScale;
 }
 
+static int gSkippedFirstMouseEventAfterCapture = 0;
+GfVec2d Editor::GetMouseDelta() {
+    ImGuiIO& io = ImGui::GetIO();
+    if(_mouseCaptured && gSkippedFirstMouseEventAfterCapture > 0) {
+        return {0, 0};
+    }
+    return {io.MouseDelta.x, io.MouseDelta.y};
+}
+
+
+void Editor::SetMouseCaptured(bool set) {

```

Comment:
`bool set` I generally avoid to name variables with names which can be confused with other things like data structures. Ideally renaming `set` to something else like `mouseCaptured` or other would be great